src/sentry/api/endpoints/organization_minimal_projects.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import EnvironmentMixin, region_silo_endpoint
+from sentry.api.bases.organization import OrganizationAndStaffPermission, OrganizationEndpoint
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.project import MinimalProjectSerializer
+from sentry.models.project import Project
+
+
+@region_silo_endpoint
+class OrganizationMinimalProjectsEndpoint(OrganizationEndpoint, EnvironmentMixin):
+    # this is for the UI only
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
+    }
+    permission_classes = (OrganizationAndStaffPermission,)
+    owner = ApiOwner.ISSUES
```

Comment:
i would like to make the owner "app-backend" or some kind of common team, but doesn't seem possible atm.

Reply:
ProjectsEndpoint is unowned and I believe this one should go to the same owner. Right now we treat unowned as  everything common ground and maintain them in the core product rotation.

I'll create a github group for that eventually and move them all there so for now feel free to remove the owner and add it to [the allowlist](https://github.com/getsentry/sentry/blob/master/src/sentry/apidocs/api_ownership_allowlist_dont_modify.py)